.github/workflows/staging.yml

```diff
@@ -73,7 +73,7 @@ jobs:
           K8S_SECRET_VENE_PAYMENTS_BAMBORA_API_KEY: ${{ secrets.GH_QA_BAMBORA_API_KEY }}
           K8S_SECRET_VENE_PAYMENTS_BAMBORA_API_SECRET: ${{ secrets.GH_QA_BAMBORA_API_SECRET }}
           K8S_SECRET_VENE_PAYMENTS_BAMBORA_PAYMENT_METHODS: ${{ secrets.GH_QA_BAMBORA_PAYMENT_METHODS }}
-          K8S_SECRET_PROFILE_API_URL: "https://profiili-api.test.kuva.hel.ninja/graphql/"
+          K8S_SECRET_PROFILE_API_URL: "https://profile-api.test.hel.ninja/graphql/"
```

Comment:
IIRC, the people in charge of migrating Profiili from Kuva to Kanslia had mentioned that the Kanslia one wasn't ready yet 🤔 

https://helsinkicity.slack.com/archives/CBA1QHTHN/p1632392257007500
<img width="366" alt="image" src="https://user-images.githubusercontent.com/15201480/140275388-6ea8a417-10d4-4688-ac7d-8771f2f2c8da.png">